mainapp/helpers.py

```diff
@@ -74,7 +74,7 @@ def get_rated_bookids(user_ratings):
     '''
     already_rated = []
     for rating in user_ratings:
-        book_id = rating.bookrating
+        book_id = rating.bookid
```

Comment:
Nice catch, so then recommendation was a bit bugged earlier xD
@maneprajakta See this [Line](https://github.com/Praful932/Kitabe/blob/master/mainapp/views.py#L67)

Reply:
Yep seems now better recommendation we will get :)